charts/workflow/requirements.yaml

```diff
@@ -45,6 +45,7 @@ dependencies:
     version: <registry-token-refresher-tag>
     repository: https://charts.deis.com/registry-token-refresher
   - name: router
+    condition: !global.experimental_native_ingress
```

Comment:
It's not very clean, agreed. The advantages are:
- works with older versions of `helm`
- matches the (ugly) way we do things in other charts

But I was off-base in suggesting we could fix everything in a unified way: it may not be feasible to convert other sub-charts to use tags & conditions in a perfectly backward-compatible fashion. So this should be an isolated decision. I know you feel strongly about it @krancour and my main concern at this point is that @kris-nova doesn't get whiplash if we want to revert the deis-router changes. 😄